lib/ansible/modules/network/meraki/meraki_switchport.py

```diff
@@ -33,6 +33,8 @@
     allowed_vlans:
         description:
         - List of VLAN numbers to be allowed on switchport.
+        - Relevant only for trunk ports.
+        - Defaults to C(all) if not specified.
```

Comment:
Only on creation, I would expect that if it's not supplied on existing objects, it would not touch the value, but keep the way it was.